common/flatpak-dir.c

```diff
@@ -8626,6 +8628,7 @@ flatpak_dir_deploy (FlatpakDir          *self,
   options.enable_fsync = FALSE; /* We checkout to a temp dir and sync before moving it in place */
   options.bareuseronly_dirs = TRUE; /* https://github.com/ostreedev/ostree/pull/927 */
   checkoutdirpath = g_file_get_path (checkoutdir);
+  checkoutdir_basename = tmp_dir_handle.path;  /* so checkoutdirpath = deploy_base_dfd / checkoutdir_basename */
```

Comment:
Doesn't seem like this variable is actually used?

Reply:
It’s used in the second commit. It’s my mistake that I committed it in the first commit rather than the second. I’ve fixed that now.